### DISCUSSION TITLE
Issues adding subtitle track

### DISCUSSION THREAD

**Discussion by Ohyah55** [mifi/lossless-cut#1623] [Q&A]
I have a video file and a subtitle track in .ass format. No matter what I try I can't get lossless cut to merge the two files. I add the subtitle track to the project and export the file and when I re-import the new file into lossless-cut I can see a 3rd track called 'bin_data [6][0][0][0]'. When I extract this track I get a .bin file with my subtitles inside. But there's no subtitle track when I open the file in any media players.

I have tried converting the subtitle file into other formats. I tried .srt and .mks with the same result. Just the extra 'bin_data' track but no actual subtitle track. The subtitle file itself should be fine because I can add it to my player as an external subtitle track and it plays fine but it'd be nice if I could merge them into one file.

One thing that's different is that the resulting .bin file does contain the subtitles, but there is no formatting at all, just a giant wall of text. The .ass, .srt and .mks files all had some formatting in the original file. I'm not sure if that matters but thought it worth mentioning.

**Comment by mifi** [comment]
That's odd. I just tried this exact thing and it works for me. I tried with an mp4 file and an .ass file from here:
https://hhsprings.bitbucket.io/docs/programming/examples/ffmpeg/subtitle/ass.html

I assume you're using the latest version of lsosslesscut?
